From abb3b388aaf7ea72ea1fcad77ead52f15a0cf543 Mon Sep 17 00:00:00 2001
From: Thomas Devoogdt <thomas@devoogdt.com>
Date: Tue, 1 Apr 2025 22:56:00 +0200
Subject: [PATCH] lib: miniz: only require a C compiler

Commit https://github.com/richgel999/miniz/commit/05ab4dc05c9d1e1f951f5849b659e2c7e291a620
"Add some catch2 tests" dropped that C flag, but should instead only have added a simple
enable_language call for the tests only.

Upstream: https://github.com/richgel999/miniz/commit/383e551cccf41fc2ce1177712d6a37a85d936642
Signed-off-by: Thomas Devoogdt <thomas@devoogdt.com>
---
 lib/miniz/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/miniz/CMakeLists.txt b/lib/miniz/CMakeLists.txt
index 859133ad0..f3d80f80a 100644
--- a/lib/miniz/CMakeLists.txt
+++ b/lib/miniz/CMakeLists.txt
@@ -7,7 +7,7 @@ if(DEFINED PROJECT_NAME)
 endif()
 
 if(CMAKE_MINOR_VERSION LESS 12)
-  project(miniz)
+  project(miniz C)
   # see issue https://gitlab.kitware.com/cmake/cmake/merge_requests/1799
 else()
   project(miniz C)
-- 
2.43.0

